### PR TITLE
Sahibnew

### PR DESCRIPTION
**Pull request by SahibS2022**

### CODE REVIEW

File: src/main/java/com/naveenautomation/Pages/CamerasPage.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.naveenautomation.Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import com.naveenautomation.Browsers.ProxyDriver;
+
+public class CamerasPage extends Page {
+
+	private static final String PAGE_URL = "product/category&path=33";
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/CamerasPage.java:10]: this URL is not correct, it should be just product/

File: src/main/java/com/naveenautomation/Pages/ChangePasswordPage.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
 public class ChangePasswordPage extends Page {
 
 	
-	private static final String PAGE_URL="/password";
+	private static final String PAGE_URL="account/password";
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/ChangePasswordPage.java:11]: if account is common in all URL, it should be in base URL and not in page url

File: src/main/java/com/naveenautomation/Pages/MyAccountPage.java [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 public class MyAccountPage extends Page {
 
-	private static final String PAGE_URL="/account";
+	private static final String PAGE_URL="account/account";
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/MyAccountPage.java:10]: if account is common in all URL, it should be in base URL and not in page url

File: src/main/java/com/naveenautomation/Pages/OrderHistoryPage.java [hunk 4]
```diff
@@ -8,7 +8,7 @@
 
 public class OrderHistoryPage extends Page {
 
-	private static final String PAGE_URL="/order";
+	private static final String PAGE_URL="account/order";
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/OrderHistoryPage.java:11]: same as above

File: src/main/java/com/naveenautomation/Pages/RegisterAccountPage.java [hunk 5]
```diff
@@ -7,7 +7,7 @@
 
 public class RegisterAccountPage extends Page {
 
-	private static final String PAGE_URL = "/register";
+	private static final String PAGE_URL = "account/register";
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/RegisterAccountPage.java:10]: same as above

File: src/test/java/com/naveenautomation/tests/CamerasTest.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+import org.testng.asserts.SoftAssert;
+
+import com.naveenautomation.Base.TestBase;
+import com.naveenautomation.Pages.AccountLoginPage;
+import com.naveenautomation.Pages.CamerasPage;
+import com.naveenautomation.Pages.MyAccountPage;
+
+public class CamerasTest extends TestBase {
+
+	AccountLoginPage accountLoginPage;
```
Comment (PivotITAcademy) [inline src/test/java/com/naveenautomation/tests/CamerasTest.java:15]: Plz add access modifier

**Review by PivotITAcademy — CHANGES_REQUESTED** [review]
Plz do the changes

File: src/main/java/com/naveenautomation/Pages/CamerasPage.java (hunk 1 above)
Comment (SahibS2022) [reply src/main/java/com/naveenautomation/Pages/CamerasPage.java:10]: URL changed as recommended.

File: src/main/java/com/naveenautomation/Pages/ChangePasswordPage.java (hunk 2 above)
Comment (SahibS2022) [reply src/main/java/com/naveenautomation/Pages/ChangePasswordPage.java:11]: URL changed as recommended.

File: src/main/java/com/naveenautomation/Pages/MyAccountPage.java (hunk 3 above)
Comment (SahibS2022) [reply src/main/java/com/naveenautomation/Pages/MyAccountPage.java:10]: URL changed as recommended.

File: src/main/java/com/naveenautomation/Pages/OrderHistoryPage.java (hunk 4 above)
Comment (SahibS2022) [reply src/main/java/com/naveenautomation/Pages/OrderHistoryPage.java:11]: URL changed as recommended.

File: src/main/java/com/naveenautomation/Pages/RegisterAccountPage.java (hunk 5 above)
Comment (SahibS2022) [reply src/main/java/com/naveenautomation/Pages/RegisterAccountPage.java:10]: URL changed as recommended.

File: src/test/java/com/naveenautomation/tests/CamerasTest.java (hunk 6 above)
Comment (SahibS2022) [reply src/test/java/com/naveenautomation/tests/CamerasTest.java:15]: Access modifier Added

**Pull request closed without merging**